### PR TITLE
Some changes to the first few pages

### PR DESCRIPTION
**Pull request by KathrynLA**

### CODE REVIEW

**Review by masinter — CHANGES_REQUESTED** [review]
Of course this still needs some work before it is ready to be merged, but good start.

File: content/en/_index.md [hunk 1]
```diff
@@ -9,25 +9,25 @@ type: docs
 
 Interlisp was a pioneering software development environment from the 1970s at Xerox PARC targeted primarily at the AI software development community. A group of its developers and users have gotten together to revive Interlisp -- as a language, environment, spirit -- in the Medley Interlisp Project.
 
-# [Project Overview](project)
+# [What is the Medley Interlisp Project?](project)
 What we are doing and why.
 
-# [About the software](about)
+# [What does Interlisp have to do with current systems?](about)
```
Comment (masinter) [inline content/en/_index.md:15]: The 'current systems' is an interesting thing to say in this section, but it's only one of many questions. "Why is Medley Interlisp worth all this effort?"

File: content/en/_index.md [hunk 2]
```diff
@@ -9,25 +9,25 @@ type: docs
...
 What are the main features that make it interesting to revive?
 How might we apply the lessons from Interlisp to current systems?
 
-# [Using Medley today](using)
+# [How do I try using Interlisp?](using)
```
Comment (masinter) [inline content/en/_index.md:19]: "There is no try, there is only do."
I think we're calling the system we're maintaining Medley, the project the Medley Interlisp Project...
But "How can I get medley?" or "How do I run Medley?" 
I want the topic to cover not only how to get it, but how to use it once you have. "How do I try it?" (OK, there is a "try" sometimes).

File: content/en/_index.md [hunk 3]
```diff
@@ -9,25 +9,25 @@ type: docs
...
 How you can try it out and what barriers you'll encounter.
 Pointers to documentation, running online, installing, reporting bugs.
 	
-# [About the Project](project)
+# [What are your goals? How do I participate?](project)
 What we're doing, how we work, what our goals are, how you can participate, how you can contribute. Pointers to github. Open Source.
 
-# [Applications](apps)
+# [How do I use existing applications built in Interlisp?](apps)
```
Comment (masinter) [inline content/en/_index.md:26]: does it "exist" until we revive it? ("I compute, therefor I am").
"What Applications build in Interlisp are there?"

This is supposed to excite people interested in the history of AI.

File: content/en/_index.md [hunk 4]
```diff
@@ -9,25 +9,25 @@ type: docs
...
 Other applications built in Interlisp that we're now able to revive as well.
 
-# [Donate](donate)
+# [How do I donate to the project?](donate)
 How and why to donate money... think of us as a virtual museum exhibit.
```
Comment (masinter) [inline content/en/_index.md:30]: and why should I? What will you do with it?

File: content/en/_index.md [hunk 5]
```diff
@@ -9,25 +9,25 @@ type: docs
...
 
-# [History](history)
+# [What materials do you have to help with research?](history)
```
Comment (masinter) [inline content/en/_index.md:32]: I suppose "help with research" is a kind-of goal, but we'll need to say more about why we're providing contemporaneous citations for finding about more.

File: content/en/project/_index.md [hunk 6]
```diff
@@ -1,5 +1,11 @@
 
-## Project overview
+# What is the Medley Interlisp Project?
+
+The Medley Interlisp Project was created to revive Medley Interlisp. 
+
+# [What does Interlisp have to do with current systems?](about)
+What are the main features that make it interesting to revive?
+How might we apply the lessons from Interlisp to current systems?
```
Comment (masinter) [inline content/en/project/_index.md:9]: is this a repeat of other sections?

File: content/en/_index.md (hunk 1 above)
Comment (stumbo) [reply content/en/_index.md:15]: Maybe, the broad overarching theme here is, why should I care?  Which leads to discussions of how Medley Interlisp has influenced today's software, how the development environment (both the software and social) was a fertile ground for experimentation and prototyping new ideas.

File: content/en/_index.md (hunk 2 above)
Comment (stumbo) [reply content/en/_index.md:19]: Larry goes Yoda.  But it's a great way at looking at it and ties back into the mindset that led to the creation of the environment.

File: content/en/project/_index.md (hunk 6 above)
Comment (stumbo) [reply content/en/project/_index.md:9]: Might it be worth (if it hasn't been done) creating a tree diagram that builds from the initial root index to the leaves showing where content will exist?   Line 6 looks like a link back to the about section.  In my mental model there should be a hierarchy that forms the overall layout of information.  You start at the root and dig deeper into content to gain more insight on specific areas.  I'm sure there be places were it makes sense to hyperlink across nodes, but the general flow should be root to leaves.

File: content/en/project/_index.md [hunk 7]
```diff
@@ -13,6 +19,11 @@ We also want to restore and present earlier versions of Interlisp, for the stude
 
 There has been a lot of cleanup and adaptation to make it usable again in the modern world. Among other developments, you can now run Medley Interlisp on many OS and hardware configurations, or at https://online.interlisp.org in the cloud, using a web browser. See our [2021 Annual Report](news/2021medleyannualreport), [Project News](/news/), and [recent release notes](https://github.com/Interlisp/medley/releases) . We've also been working on integration of the Interlisp style development with git and GitHub, Docker and other modern components.
 
+# [How do I try using Interlisp?](using)
```
Comment (stumbo) [inline content/en/project/_index.md:22]: Ia Using a subsection of Project?  If so, this belongs here, if not it should be moved.

**Review by stumbo — CHANGES_REQUESTED** [review]

**Comment by stumbo** [comment]
Kathryn - It looks like you're well on your way to mastering the Git workflow.  This is a great start and an excellent way to put ideas out for comment.

**Pull request merged**